### PR TITLE
Adds sandbox uri

### PR DESCRIPTION
**Pull request by skonto**
By default enable the sandbox uri. Check at the bottom of the image the uri.

image

### CODE REVIEW

**Comment by skonto** [comment]
@ArtRand @susanxhuynh pls review.

File: docker/runit/service/spark/run [hunk 1]
```diff
@@ -53,6 +53,7 @@ function configure_properties() {
     add_if_non_empty spark.ssl.trustStorePassword "${SPARK_SSL_TRUSTSTOREPASSWORD}"
     add_if_non_empty spark.ssl.protocol "${SPARK_SSL_PROTOCOL}"
     add_if_non_empty spark.ssl.enabledAlgorithms "${SPARK_SSL_ENABLEDALGORITHMS}"
+    echo "spark.mesos.proxy.baseURL=../../mesos" >> /opt/spark/dist/conf/mesos-cluster-dispatcher.properties
```
Comment (susanxhuynh) [inline docker/runit/service/spark/run]: Could you avoid hardcoding the filename (/opt/spark/dist/conf/mesos-cluster-dispatcher.properties)? Maybe store it in a variable?

**Comment by skonto** [comment]
@susanxhuynh pls review, clear for merge I think.

**Comment by ArtRand** [comment]
retest this please

**Comment by skonto** [comment]
@ArtRand whats wrong with the internal sanity test?

**Comment by susanxhuynh** [comment]
The "teamcity/sanity-test" was added by the infrastructure team to test something out. They said we could ignore it for now.

**Pull request merged**